src/features/people/PeopleDetails/index.js

```diff
@@ -18,20 +25,94 @@ const PersonDetails = () => {
 
     const details = useSelector(selectDetails);
     const status = useSelector(selectStatus);
+    const actorCast = useSelector(selectPersonCast);
+    const actorCrew = useSelector(selectPersonCrew);
+    const [genres, setGenres] = useState([]);
+
+    useEffect(() => {
+        const fetchGenres = async () => {
+            const genresData = await getGenreList();
+            setGenres(genresData.genres);
+        };
+        fetchGenres();
+    }, []);
+
+    const nameGenres = (genreIds) => {
+        if (!genres || genres.length === 0) {
+            return [];
+        }
+
+        return genreIds.map((id) => {
+            const genre = genres.find((genre) => genre.id === id);
+            return genre ? genre.name : '';
+        });
+    };
+
+    const getReleaseYear = (releaseDate) => {
+        const date = new Date(releaseDate);
+        return date.getFullYear();
+    };
+
+    const formatVote = (vote) => vote.toLocaleString(
```

Comment:
Here You can use my (or maybe find a better one 🙏🏼 ) solution for rates without decimals.

Reply:
Thanks for your suggestion! 😊 I'll check out your solution for rates without decimals and incorporate it in my next PR. Appreciate your input! 🚀👍🏼